### PR TITLE
Add iptables command to open relevant ports for Minio

### PR DESCRIPTION
**Pull request by nitisht**
## Description
Add `iptable` command for RHEL/CentOS deployments.

## Motivation and Context
See #5022

## How Has This Been Tested?
Manually on CentOS7

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added unit tests to cover my changes.
- [ ] I have added/updated functional tests in [mint](https://github.com/minio/mint). (If yes, add `mint` PR # here: )
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by balamurugana** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-xl.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-dist-xl.sh | :heavy_check_mark: |
| mint-gateway-s3.sh | :heavy_check_mark: |
| mint-gateway-azure.sh | :heavy_check_mark: |

File: README.md [hunk 1]
```diff
@@ -52,6 +52,14 @@ chmod +x minio
 ./minio server /data
 ```
 
+### iptables in RHEL/CentOS
```
Comment (harshavardhana) [inline README.md]: Could we also add `ufw` i.e Ubuntu/Debian as well? - this section can be about firewall not specific to a platform.

**Review by harshavardhana — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (nitisht) [reply README.md]: good idea. will do

Comment (deekoder) [reply README.md]: good idea. I concur with harsha. I love digital oceans guides on these topics.

**Review by deekoder — CHANGES_REQUESTED** [review]

**Comment by nitisht** [comment]
PR is updated, PTAL @harshavardhana @deekoder

**Review by harshavardhana — DISMISSED** [review]
LGTM

**Comment by balamurugana** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-xl.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-dist-xl.sh | :heavy_check_mark: |
| mint-gateway-s3.sh | :heavy_check_mark: |
| mint-gateway-azure.sh | :heavy_check_mark: |

**Comment by balamurugana** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-xl.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-dist-xl.sh | :heavy_check_mark: |
| mint-gateway-s3.sh | :heavy_check_mark: |
| mint-gateway-azure.sh | :heavy_check_mark: |

**Review by harshavardhana — APPROVED** [review]

**Review by deekoder — APPROVED** [review]

**Pull request merged**